Content.Client/GameObjects/Components/Atmos/GasAnalyzerMenu.cs

```diff
@@ -149,20 +149,21 @@ public void Populate(GasAnalyzerBoundUserInterfaceState state)
             {
                 _statusContainer.AddChild(new Label
                 {
-                    Text = Loc.GetString("Error: {0}", state.Error),
+                    Text = Loc.GetString("gas-analyzer-error", ("error", state.Error)),
                     FontColorOverride = Color.Red
                 });
                 return;
             }
 
             _statusContainer.AddChild(new Label
             {
-                Text = Loc.GetString("Pressure: {0:0.##} kPa", state.Pressure)
+                Text = Loc.GetString("gas-analyzer-pressure", ("pressure", state.Pressure))
             });
             _statusContainer.AddChild(new Label
             {
-                Text = Loc.GetString("Temperature: {0:0.#}K ({1:0.#}°C)", state.Temperature,
-                    TemperatureHelpers.KelvinToCelsius(state.Temperature))
+                Text = Loc.GetString("gas-analyzer-temperature",
+                            ("kelvin", state.Temperature),
+                            ("celsius", TemperatureHelpers.KelvinToCelsius(state.Temperature)))
```

Comment:
Turning kelvin to celcius should be handled by the localization system itself, with a custom function.